templates/sponsors/sponsor.html

```diff
@@ -1,50 +1,42 @@
 {% set main_class='sponsors' %}
 {% extends "base.html" %}
-{% block title %}Sponsor EMF{% endblock %}
+{% block title %}Sponsor Electromagnetic Field{% endblock %}
 {% block body %}
 
-<h2>Sponsor EMF</h2>
+<h2>Sponsor Electromagnetic Field</h2>
 
-<div class="well">
-    Electromagnetic Field is no longer accepting sponsors for our 2016 event.
-</div>
     <img src="{{ url_for('static', filename='images/sponsor/sponsor_1.jpg')}}"
-            class="img-thumbnail img-responsive" style="float:right; width:320px">
+            class="img-thumbnail img-responsive" style="float:right; width:320px; margin-left: 16px;">
 <p>Electromagnetic Field (EMF) is the UK’s largest technology-focused camping festival: A three-day event for people with an inquisitive mind and an interest in science, engineering, arts, technology, crafts, games, DIY, or security.</p>
```

Comment:
"camping festival: A three-day" either the "a" should be lower case or a new sentence